genrl/trainers/her_trainer.py

```diff
@@ -0,0 +1,78 @@
+from typing import Type, Union
+
+from genrl.trainers import OffPolicyTrainer
+
+
+class HERTrainer(OffPolicyTrainer):
```

Comment:
Might need to refactor this -- arent there functions here which are exact duplicates of the ones in `OffPolicyTrainer`?